### ISSUE TITLE
Error: Cannot find module 'csp_evaluator'

### ISSUE THREAD

**Issue by frodesigns** [GoogleChrome/lighthouse-ci#662]
**Describe the bug**
This morning our Lighthouse runs started failing with the following error: 
`Runtime error encountered: Cannot find module 'csp_evaluator/dist/lighthouse/lighthouse_checks.js'`

**Environment:**
 - OS: ubuntu-latest
 - Node: 12.x
 - Lighthouse: 0.8.x

**Full error log:**
```bash
Run lhci autorun
✅  .lighthouseci/ directory writable
✅  Configuration file found
✅  Chrome installation found
✅  GitHub token set
Healthcheck passed!

Automatically determined ./build as `staticDistDir`.
Set it explicitly in lighthouserc.json if incorrect.

Started a web server on port 41339...
Running Lighthouse 3 time(s) on http://localhost:41339/index.html
Run #1...failed!
Error: Lighthouse failed with exit code 1
    at ChildProcess.<anonymous> (/opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/src/collect/node-runner.js:120:21)
    at ChildProcess.emit (events.js:314:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
Wed, 04 Aug 2021 14:28:27 GMT ChromeLauncher Waiting for browser.
Wed, 04 Aug 2021 14:28:27 GMT ChromeLauncher Waiting for browser...
Wed, 04 Aug 2021 14:28:27 GMT ChromeLauncher Waiting for browser.....
Wed, 04 Aug 2021 14:28:27 GMT ChromeLauncher Waiting for browser.....✓
Wed, 04 Aug 2021 14:28:27 GMT ChromeLauncher Killing Chrome instance 3117
Runtime error encountered: Cannot find module 'csp_evaluator/dist/lighthouse/lighthouse_checks.js'
Require stack:
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/lib/csp-evaluator.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/audits/csp-xss.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/config/config-helpers.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/config/config.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/index.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-cli/commands/list-audits.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-cli/commands/commands.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-cli/bin.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-cli/index.js
Error: Cannot find module 'csp_evaluator/dist/lighthouse/lighthouse_checks.js'
Require stack:
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/lib/csp-evaluator.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/audits/csp-xss.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/config/config-helpers.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/config/config.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/index.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-cli/commands/list-audits.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-cli/commands/commands.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-cli/bin.js
- /opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-cli/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/opt/hostedtoolcache/node/12.22.4/x64/lib/node_modules/@lhci/cli/node_modules/lighthouse/lighthouse-core/lib/csp-evaluator.js:14:5)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)

Error: Process completed with exit code 1.
```

**Comment by brendankenny** [comment]
this is https://github.com/google/csp-evaluator/issues/36

manually adding `csp_evaluator@1.0.1` as a dep should work for now.

**Comment by janosh** [comment]
Same problem here https://github.com/janosh/afara/runs/3243240303.

**Comment by frodesigns** [comment]
Issue is resolved with google/csp-evaluator#36
